lib/datadog/ci.rb

```diff
@@ -1,5 +1,4 @@
 # typed: strict
-require 'ddtrace'
 
 module Datadog
```

Comment:
This one is needed and should not be removed as it's used for the CI Visibility features (as documented in  https://docs.datadoghq.com/continuous_integration/setup_tests/ruby/?tab=rspec ). It does not actually get required from ddtrace.rb :)